bindings/python/gnucash_core.py

```diff
@@ -28,6 +28,8 @@
 #  @author Jeff Green,   ParIT Worker Co-operative <jeff@parit.ca>
 #  @ingroup python_bindings
 
+from enum import IntEnum
+
```

Comment:
Why the blank line?

Reply:
I was following [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports should be grouped in the following order:
>
>   1. Standard library imports.
>   2. Related third party imports.
>   3. Local application/library specific imports.
>
>_You should put a blank line between each group of imports._

(italics inserted by me)

Enum is a standard library (group 1) while the following gnucash imports are local/application specific (group 3).